labgrid/config.py

```diff
@@ -5,6 +5,7 @@
 """
 import os
 import warnings
+from typing import Dict
```

Comment:
Does Labgrid sort its imports?

Reply:
No, I suppose I just don't understand how the ordering works

Perhaps 'warnings' is consider an internal module and 'typing' is not?